Bash_CLI/src/Token/Token.py

```diff
@@ -0,0 +1,33 @@
+class Token(object):
+    """
+    Class, that store data-string and some useful information about it:
+    separator, that follows data-string, and quotation, that wraps data-string.
+    """
+
+    def __init__(self, data: str, separator_after_str: str, quotation: str):
+        self._data = data
+        self._separator_after_str = separator_after_str
+        self._quotation = quotation
+    
+    def __str__(self) -> str:
+        return self._quotation + self._data + self._quotation + self._separator_after_str
+    
+    def empty(self) -> bool:
+        """Check that data-string is empty."""
+        return len(self._data) == 0
+    
+    def set_data(self, data: str):
+        """Set the data-string."""
+        self._data = data
+
+    def data(self) -> str:
+        """Return the data-string."""
+        return self._data
+    
+    def set_separator(self, sep: str):
```

Comment:
https://stackoverflow.com/questions/2627002/whats-the-pythonic-way-to-use-getters-and-setters